scripts/nrfconnect-license.ts

```diff
@@ -142,7 +142,11 @@ const allSourceFiles = () =>
             '*.sass',
         ],
         {
-            ignore: [...entriesInGitignore, '.git'],
+            ignore: [
+                ...entriesInGitignore,
+                'scripts/nordic-publish.js',
```

Comment:
Is this added because the license is not put in the very top of the file?

Reply:
There is code `"use strict"` before the block with the license. So yes. Might alternatively change the script to look through the entire file for the license.

Reply:
We could also change the script to also skip the `"use strict"`. But so far, this is the only exception and it is fine to just exclude it this way here.